lib/page_util.rb

```diff
@@ -43,6 +37,18 @@ def push_all_resource_pages(keep_existing_phrases)
     end
   end
 
+  def push_name_description
+    phrases = { name: @resource.name, description: @resource.description }
+    push_page(phrases, 'name_description.xml', false)
+  end
+
+  def push_translatable_attributes
+    phrases = {}
+    @resource.resource_attributes.where(is_translatable: true).each { |a| phrases[a.key] = a.value }
```

Comment:
There's a slick ruby syntactical sugar way to do this:
```ruby
phrases = Hash[@resource.resource_attributes.where(is_translatable: true).pluck(:key, :value)
```
The `pluck` method selects just the columns listed, and loads them into an array of arrays (instead of `ResourceAttribute` objects). `Hash[]` converts an array of arrays into a hash using the first value of each pair as the key and the second value as the value.